doc/kios.md

```diff
@@ -1 +1,726 @@
 # Kios Module
+Module | HTTP Method | URL | Description 
+--- | --- | --- | ---
+[Add](#add) | POST | /kios | Add Data Kios
+[Edit](#edit) | PUT | /kios/:id | Edit Data Kios
+[View](#view) | GET | /kios/:id | View Data Kios
+[Delete](#delete) | DELETE | /kios/:id | Delete Data Kios
+
+## <a name="add"></a>Add Data Kios
+
+### Endpoint
+POST /kios
+
+## Database
+![](./kios-layer.png)
+
```

Comment:
- you have 2 foreign key to other table : id and code. You should use one foreignKey, so remove code foreign key like branch_code, company-code, and warehouse_code.
- field of kunci. can you explain detail the field? documentation readers don't know what this is for